python/ray/data/_internal/execution/streaming_executor.py

```diff
@@ -306,15 +298,11 @@ def _scheduling_loop_step(self, topology: Topology) -> bool:
         for op_state in topology.values():
             op_state.refresh_progress_bar()
 
+        self._update_stats_metrics(state="RUNNING")
+        _log_op_metrics(topology)
```

Comment:
let's also add a 5s interval for `_log_op_metrics` and `_debug_dump_topology`. I found it a bit too verbose.